clang/include/clang/APINotes/APINotesManager.h

```diff
@@ -159,7 +159,8 @@ class APINotesManager {
   ArrayRef<APINotesReader *> getCurrentModuleReaders() const {
     bool HasPublic = CurrentModuleReaders[ReaderKind::Public];
     bool HasPrivate = CurrentModuleReaders[ReaderKind::Private];
-    assert((!HasPrivate || HasPublic) && "private module requires public module");
+    assert((!HasPrivate || HasPublic) &&
+           "private module requires public module");
```

Comment:
Please don't change untouched codes.

Reply:
clang-format did that when I touched the file. Does the formatting requirement apply to a narrower set of changes than the entire file?

Reply:
Yes, we prefer to format the changed line only. Otherwise the backporting may be problematic. And git blaming will be harder.

One possible way maybe:

> git diff -U0 --no-color --relative HEAD^ | clang/tools/clang-format/clang-format-diff.py -p1 -i

Reply:
OK, I set my VS Code to just format changed lines, looks like I have it working now.